### DISCUSSION TITLE
Multiple Molecule + "concentration" Regression Task

### DISCUSSION THREAD

**Discussion by kamilm** [chemprop/chemprop#746] [Q&A] [answered]
Hello, 

First, thank you all for the work and open sourcing such a robust and powerful model architecture/package. 

I have a regression task where I my inputs are molecules and concentrations, I am curious if this is within the scope of ChemProp? and if so, where in the docs can I read about implementing this? I'm not above misusing the package and 
hacking something together, I'm just not sure where to look in the package so that I can couple "molecules" and a "property" 😄 .

My data and problem statement are as such: 

`molecule-A` @ `conc-A1` + `molecule-B` @ `conc-B1` + `molecule-C` @ `conc-C1` gives `result-1`
`molecule-A` @ `conc-A2` + `molecule-B` @ `conc-B2` + `molecule-C` @ `conc-C2` gives `result-2`
`molecule-A` @ `conc-A3` + `molecule-B` @ `conc-B3` + `molecule-C` @ `conc-C3` gives `result-3`
...
`molecule-D` @ `conc-D1` + `molecule-E` @ `conc-E1` + `molecule-F` @ `conc-F1` gives `result-M`
...
`molecule-X` @ `conc-Xn` + `molecule-Y` @ `conc-Yn` + `molecule-Z` @ `conc-Zn` gives `result-N`

I would like to be able to pass in:

`new-molecule-A` @ `new-conc-A` + `new-molecule-B` @ `new-conc-B` + `new-molecule-C` @ `new-conc-C`

and have the output be some continuous value. 

Thank You, I hope this was clear. I can provide further clarification if needed. 

-Kamil

**Comment by kevingreenman** [comment, marked as answer]
Hi @kamilm, apologies for the slow reply. This functionality is not currently available in any version of chemprop, but we have been working on projects related to mixed solvents and areas like that where this would be useful, and this may become a part of chemprop in the future.
